libs/@guardian/source-react-components/src/text-area/TextArea.tsx

```diff
@@ -98,27 +104,37 @@ export const TextArea = ({
 
 		return undefined;
 	};
+	const mergedTheme = { ...themeTextArea, ...theme };
```

Comment:
This component doesn't have any existing support for `ThemeProvider` theming so there's no need to use the shared `mergeThemes` method here.

Reply:
Nice! yea this is what i need to do with the Icons also